readthedocs/templates/account/email.html

```diff
@@ -9,68 +9,104 @@
+      <fieldset class="blockLabels">
+
+        {% for emailaddress in user.emailaddress_set.all %}
+          <div class="ctrlHolder">
+            <label for="email_radio_{{forloop.counter}}" class="{% if emailaddress.primary %}primary_email{%endif%}">
+              <input id="email_radio_{{forloop.counter}}" type="radio" name="email" {% if emailaddress.primary %}checked="checked"{%endif %} value="{{emailaddress.email}}"/>
+
+              {{ emailaddress.email }}
+                {% if emailaddress.verified %}
+                  <span class="verified">{% trans "Verified" %}</span>
+                {% else %}
+                  <span class="unverified">{% trans "Unverified" %}</span>
+                {% endif %}
+                {% if emailaddress.primary %}<span class="primary">{% trans "Primary" %}</span>{% endif %}
+            </label>
+          </div>
+        {% endfor %}
+
+        <div class="buttonHolder">
+          <button class="secondaryAction" type="submit" name="action_primary" >{% trans 'Make Primary' %}</button>
+          <button class="secondaryAction" type="submit" name="action_send" >{% trans 'Re-send Verification' %}</button>
+          <button class="primaryAction" type="submit" name="action_remove" >{% trans 'Remove' %}</button>
+        </div>
+
+      </fieldset>
+    </form>
 
-</fieldset>
-</form>
+  {% else %}
 
-{% else %}
-<p><strong>{% trans 'Warning:'%}</strong> {% trans "You currently do not have any email address set up. You should really add an email address so you can receive notifications, reset your password, etc." %}</p>
+  <p><strong>{% trans 'Warning:'%}</strong>{% trans "You currently do not have any email address set up. You should really add an email address so you can receive notifications, reset your password, etc." %}</p>
 
-{% endif %}
+  {% endif %}
 
+  <h2>{% trans "Add E-mail Address" %}</h2>
 
-    <h2>{% trans "Add E-mail Address" %}</h2>
-
-    <form method="post" action="{% url 'account_email' %}" class="add_email">
-        {% csrf_token %}
-        {{ form.as_p}}
-        <button name="action_add" type="submit">{% trans "Add E-mail" %}</button>
-    </form>
+  <form method="post" action="{% url 'account_email' %}" class="add_email">
+    {% csrf_token %}
+    {{ form.as_p }}
+    <button name="action_add" type="submit">{% trans "Add E-mail" %}</button>
+  </form>
 
 {% endblock %}
 
+{% block footerjs %}
+
+  (function() {
+    var message = "{% trans 'Do you really want to remove the selected email address?' %}";
+    var actions = document.getElementsByName('action_remove');
+    if (actions.length) {
+      actions[0].addEventListener("click", function(e) {
+        if (! confirm(message)) {
+          e.preventDefault();
+        }
+      });
+    }
+  })();
+
+  (function() {
+    var primaryButton = document.querySelector("button[name=action_primary]");
+    var sendVerificationButton = document.querySelector("button[name=action_send");
+
+    {% if user.emailaddress_set.all %}
```

Comment:
nit: use `.exists` here instead of `.all`.

Reply:
Thank you.
I have pushed the changes.